### ISSUE TITLE
Confusing remediation recommendation in aXe

### ISSUE THREAD

**Issue by goodwitch** [dequelabs/axe-core#742]
We've recently discovered a confusing remediation recommendation in aXe that is automatically created for the following type of issue:

> Auto Issue Text:  
> Links must be distinguished from surrounding text in a way that does not rely on color

And just below that...aXe says:

> Recommendation to fix
> Fix all of the following:
>   Links can not be distinguished from surrounding text in a way that does not rely on color

These way the recommendation fix is written is confusing.  The last sentence is written in the double negative and is unnecessarily hard understand.  We suggest changing to read something like: 

> Recommendation to fix
> Links need to be identified in some way other than just by color alone.
> 

Thanks for consider this request!
Love,
Goodwitch,  Jenn, Melanie and Susan

**Comment by luckyfordogs** [comment]
Yes. This type of clarity is paramount as aXe is used by many for whom ESL 

Mike Farrell
847.420.3900

**Comment by WilcoFiers** [comment]
How about this:

> Links need to be distinguished from surrounding text in some way other than by color

**Comment by goodwitch** [comment]
@WilcoFiers 

I love your new wording.  +1
